06-building-a-program-start-to-finish/writing-tests/output/InvertedIndex.spec.js

```diff
@@ -0,0 +1,278 @@
+/* eslint-disable no-undef */
+const invertedIndex = new InvertedIndex();
+const adventureBooks = require('../books/adventureBooks.json');
+const emptyBook = require('../books/emptyBook.json');
+const wrongFormat = require('../books/wrongFormat.json');
+
+const uploads = {
+  'adventureBooks.json': { content: adventureBooks },
+  'emptyBook.json': { content: emptyBook },
+  'wrongFormat.json': { content: wrongFormat },
+  'badfile.json': { content: badfile }
```

Comment:
hello @andela-moe the line has been deleted because i could not require such file in javascript. it will break the codes because it is a syntax error